security/integration/jersey/src/main/java/io/helidon/security/integration/jersey/SecurityFilterCommon.java

```diff
@@ -344,7 +345,11 @@ protected void abortRequest(SecurityFilter.FilterContext context,
             updateHeaders(responseHeaders, responseBuilder);
         }
 
-        if (featureConfig.isDebug()) {
+        // Run security response mappers if available, or revert to old logic for compatibility
+        ServiceLoader<SecurityResponseMapper> loader = ServiceLoader.load(SecurityResponseMapper.class);
```

Comment:
The service loader should be used only  once, and in a static section.
Also please use `HelidonServiceLoader` as it supports prioritization and some basic control through system properties.

Reply:
Yes, good point